src/Ownership/Repositories/Search/DBALOwnershipSearchRepository.php

```diff
@@ -116,6 +116,16 @@ public function searchTotal(SearchQuery $searchQuery): int
             ->fetchOne();
     }
 
+    public function doesUserForOrganisationExist(Uuid $organizerId, string $ownerId): bool
```

Comment:
Why not use the existing `search` method?

Reply:
I found this convenience method much easier to read, it communicates intent better than the generic search method.
I would prefer to keep the method, but I can move it to a separate class if you prefer?